base/applications/kbswitch/kbswitch.c

```diff
@@ -1028,58 +1078,116 @@ KbSwitch_OnPenIconMsg(HWND hwnd, UINT uMouseMsg)
     POINT pt;
     GetCursorPos(&pt);
 
-    ERR("g_hwndLastActive: %p\n", g_hwndLastActive);
+    // Get target window
+    TRACE("g_hwndLastActive: %p\n", g_hwndLastActive);
     HWND hwndTarget = GetTargetWindow(g_hwndLastActive);
-    ERR("hwndTarget: %p\n", hwndTarget);
+    TRACE("hwndTarget: %p\n", hwndTarget);
 
+    // Get default IME window
     HWND hwndIme = ImmGetDefaultIMEWnd(hwndTarget);
     if (!hwndIme)
     {
         WARN("No default IME\n");
         return;
     }
 
+    // Get IME context from another process
     HIMC hIMC = (HIMC)SendMessage(hwndIme, WM_IME_SYSTEM, IMS_GETCONTEXT, (LPARAM)hwndTarget);
     if (!hIMC)
     {
         WARN("No HIMC\n");
         return;
     }
 
+    // Workaround of TrackPopupMenu's bug
```

Comment:
"of" -> "for".
Is this a bug that happens only in ReactOS? Or a bug/"feature" that happens on Windows as well?